etc/systemd/system-generators/zfs-mount-generator.in

```diff
@@ -2,6 +2,7 @@
 
 # zfs-mount-generator - generates systemd mount units for zfs
 # Copyright (c) 2017 Antonio Russo <antonio.e.russo@gmail.com>
+# Copyright (c) 2020 InsanePrawn <insane.prawny@gmail.com>
 #
```

Comment:
Please follow the same format as the CDDL header for this, and put the copyright references after the copyright header.

It might not be standard for this licence header, but it make the code more readable (and legally it doesn't change anything.)

Reply:
I don't think it matters where they are, but I'd give the tie to above, since this license header specifically references "The above copyright notice".